### PR TITLE
Added max price per square filter

### PR DESCRIPTION
**Pull request by Shoggomo**
Added a max_price_per_square filter and added it to the config. It's probably more interesting when searching for something to buy.

### CODE REVIEW

File: flathunter/filter.py [hunk 1]
```diff
@@ -123,6 +123,21 @@ def is_interesting(self, expose):
...
+    """Exclude exposes above a given price per square"""
+
+    def __init__(self, max_pps):
+        self.max_pps = max_pps
+
+    def is_interesting(self, expose):
+        """True if price per square is below max price per square"""
+        size = ExposeHelper.get_size(expose)
+        price = ExposeHelper.get_price(expose)
+        pps = price / size
```
Comment (codders) [inline flathunter/filter.py:137]: If size or price is None, this division is going to fail. Can you move the check above?

**Review by codders — CHANGES_REQUESTED** [review]
Looks really good - thanks for following the conventions there. If you can fix the divide-by-none I'll merge it.

File: flathunter/filter.py (hunk 1 above)
Comment (Shoggomo) [reply flathunter/filter.py]: Done.

Comment (codders) [reply flathunter/filter.py]: Great!

**Pull request merged**